react-client/src/models.js

```diff
@@ -324,7 +324,7 @@ export class DataRepository extends Model {
   set title(val) { this.setData("host.title", val); }
 
   get url() { return this.getData("host.url", ""); }
-  set url(val) { this.setData("host.url", val); }
+  set url(val) { this.setData("host.url", val?.trim()); }
```

Comment:
@briri , you shouldn't need to use optional chaining in front of the "trim()" method, because JavaScript automatically handles the case where the value is null.

Reply:
Ah. I did not realize that. I just updated

Reply:
@briri thanks for making the change.